static/app/views/organizationGroupDetails/groupMerged/mergedItem.tsx

```diff
@@ -70,42 +66,56 @@ class MergedItem extends React.Component<Props, State> {
   }
 
   handleToggle = () => {
-    const {disabled, fingerprint, event} = this.props;
+    const {fingerprint} = this.props;
+    const {latestEvent} = fingerprint;
 
-    if (disabled || this.state.busy) {
+    if (this.state.busy) {
       return;
     }
 
     // clicking anywhere in the row will toggle the checkbox
-    GroupingActions.toggleUnmerge([fingerprint, event.id]);
+    GroupingActions.toggleUnmerge([fingerprint.id, latestEvent.id]);
   };
 
   handleCheckClick() {
     // noop because of react warning about being a controlled input without `onChange`
     // we handle change via row click
   }
 
+  renderFingerprint(id: string | undefined, label?: string) {
+    if (!label) {
+      return id;
+    }
+
+    return (
+      <Tooltip title={id}>
```

Comment:
should we render the tooltip component if `id` is `undefined`?

Reply:
it can practically never happen I think

I would not go any further to polish this, it's behind a featureflag and will be thrown away anyway